stripe/test/test_resources.py

```diff
@@ -823,6 +823,66 @@ def test_create_with_source_param(self):
         )
 
 
+class DisputeTest(StripeResourceTest):
+
+    def test_list_all_disputes(self):
+        stripe.Dispute.all(created={'lt': NOW})
+
+        self.requestor_mock.request.assert_called_with(
+            'get',
+            '/v1/disputes',
+            {
+                'created': {'lt': NOW},
+            }
+        )
+
+    def test_create_dispute(self):
+        stripe.Dispute.create(idempotency_key='foo', **DUMMY_DISPUTE)
+
+        self.requestor_mock.request.assert_called_with(
+            'post',
+            '/v1/disputes',
+            DUMMY_DISPUTE,
+            {'Idempotency-Key': 'foo'},
+        )
+
+    def test_retrieve_dispute(self):
+        stripe.Dispute.retrieve('dp_test_id')
+
+        self.requestor_mock.request.assert_called_with(
+            'get',
+            '/v1/disputes/dp_test_id',
+            {},
+            None
+        )
+
+    def test_update_dispute(self):
```

Comment:
Updating with evidence seems like it would be the more common case, might be better to test that instead?